### ISSUE TITLE
Error: Extension GL_VERSION_1_2 could not be loaded

### ISSUE THREAD

**Issue by boydcpeters** [tix209/SOAX#7]
I am new to SOAX and trying to run the program following the user manual, but I get an error when loading an image.

After running the 'soax_v3.7.0.exe' file, I get the following error when opening up a .tiff file:

ERROR: In ..\..\..\VTK-6.2.0\VTK-6.2.0\Rendering\OpenGL\vtkOpenGLExtensionManager.cxx, line 757
vtkOpenGLExtensionManager (0000000003BF8330): Extension GL_VERSION_1_2 could not be loaded.

ERROR: In ..\..\..\VTK-6.2.0\VTK-6.2.0\Rendering\OpenGL\vtkOpenGLExtensionManager.cxx, line 757
vtkOpenGLExtensionManager (0000000003BF8330): Extension GL_VERSION_1_2 could not be loaded.

Does anyone know how to fix this error?

My laptop has the following specifications:

Processor: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
Video Card: Quadro P1000
Video Card #2: Intel(R) UHD Graphics 630
Operating System: Windows 10
RAM: 16 GB

**Comment by vavylonis** [comment]
Hello. Is this message causing SOAX to crash or is it just a message printed out and you can still work with SOAX?

**Comment by boydcpeters** [comment]
First of all, thank you for the swift response.

Initially, I suspected that SOAX was crashing due to my image stack appearing entirely black. However, upon closer inspection and twiddling around with some of the settings, I realized that the issue lay in the low contrast of the z-stack itself. Furthermore, it turned out that I hadn't adjusted the slice location as I thought I was doing.

Everything seems to work smoothly now, so my apologies for the unnecessary issue. I've managed to conduct some preliminary tests without fine-tuning any of the parameters, and I am pleased with the initial results. Thank you for developing such a great tool.

I will close the issue as SOAX works smoothly even though it throws the OpenGL error.
